packages/editor-ui/src/plugins/codemirror/completions/utils.ts

```diff
@@ -239,7 +244,7 @@ export const applyBracketAccessCompletion = (
 
 export const hasRequiredArgs = (doc?: DocMetadata): boolean => {
 	if (!doc) return false;
-	const requiredArgs = doc?.args?.filter((arg) => !arg.name.endsWith('?')) ?? [];
+	const requiredArgs = doc?.args?.filter((arg) => !arg.name.endsWith('?') && !arg.optional) ?? [];
```

Comment:
Why the two conditions? Can there be an arg `name` that ends with `?` but has `optional` missing or `false`?

Reply:
Yes all old methods have the name ending in `?`, but I started adding `optional` to have more flexibility in rendering, I want to end up with only the `optional` check but hard to get all of them at once